auto_nag/scripts/configs/team_managers.json

```diff
@@ -0,0 +1,22 @@
+{
+  "fallback": "Andrew Overholt",
```

Comment:
Maybe it should be Thomas Elin here, @marco-c wdyt ?

Reply:
Yes

Reply:
Done in ca146caf9ec7dfa1b3f2ef0a39daeeaa3bcdc0d6